packages/kit/src/exports/vite/dev/index.js

```diff
@@ -50,11 +50,27 @@ export async function dev(vite, vite_config, svelte_config) {
 	/** @type {Error | null} */
 	let manifest_error = null;
 
+	/** @param {any} err */
+	function outputModuleLoadError(err) {
+		const msg = buildErrorMessage(err, [colors.red(`Internal server error: ${err.message}`)]);
+
+		vite.config.logger.error(msg, {
+			error: err
+		});
+		vite.ws.send({
+			type: 'error',
+			err: err
+		});
+	}
+
 	/** @param {string} id */
 	async function resolve(id) {
 		const url = id.startsWith('..') ? `/@fs${path.posix.resolve(id)}` : `/${id}`;
 
-		const module = await vite.ssrLoadModule(url);
+		let module = await vite.ssrLoadModule(url).catch((error) => {
```

Comment:
wow, this is fantastic! I didn't know about `buildErrorMessage`

I see another half dozen places where `ssrLoadModule` is called. Maybe we also need this same code in those places? We could make a utility function that calls `ssrLoadModule`

Reply:
Pulling that rebase commit back to my local took me way longer than it should have, hah.

I'll look at the other locations tomorrow.

Reply:
I added it to the other 2 places where we're loading user modules, 
for the ones generated by Kit, there's already a generic error page with the stack trace that's rendered, probably don't need it there.